test/evergreen/tsan_warnings.supp

```diff
@@ -29,7 +29,10 @@ race:__ref_track_state
 race:__wt_configure_method
 
 # FIXME-WT-13074 This Work has been split out into a separate ticket to keep PR sizes down.
-race:__wt_calloc
+race:__wt_free_int
```

Comment:
The warnings were revealed by the `WT_INTL_INDEX_*` fix. I'll address them in WT-13074